### PR TITLE
VPN market expansion (Fixes #15262)

### LINKED ISSUE #15262: Mozilla VPN expansion
## Description

Docs are Mozilla access only:

Product tracker: https://docs.google.com/spreadsheets/d/1jzLj9MJJObGBUvpGmfVsXUMjC9yTaCDNngcmWAUP668/edit?usp=sharing
Country selection: https://docs.google.com/spreadsheets/d/1mQ6QjKRdF6DgJ6FvzUKR5-K-C-8TQNVgucKJV0keJ9Y/edit?usp=sharing
Figma designs: https://www.figma.com/design/nm2nApv3ghovOu4w49ckk8/VPN-Landing-Page-(2023)?node-id=2026-2236&m=dev&t=M8GTPaDS0F28zhVk-1

---

#### Success Criteria
- [ ] Strings need to be submitted for L10n asap.
- [ ] Needs to be on dev for QA by Friday Oct 18th.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Enables some new countries for VPN availability

## Significant changes and points to review

Each of these new countries are only eligible to purchase a VPN subscription via either the Apple App Store, or Google Play Store - but NOT through FxA / Subplat on desktop. Therefore a new UI is shown for these countries in place of the usual pricing section. The new UI features a QR Code to scan on desktop, and direct links to the app stores on mobile.

To make it a little bit more complicated still, there are a very small number of countries where the Apple App Store is not currently available. Here only links to the Google Play Store are shown.

I tried to add tests to cover this all this logic. Everything is behind a switch called `VPN_WAVE_VII`

## Issue / Bugzilla link

#15262

## Testing

Test with switch on: 

```
./manage.py waffle_switch --create VPN_WAVE_VII on
```

Both app stores available in country:

- http://localhost:8000/en-US/products/vpn/?geo=AU
- http://localhost:8000/en-US/products/vpn/pricing/?geo=AU

Things to test:

- [x] Mobile subscription only UI should be displayed for new country codes.
- [x] Scanning QR code should redirect to appropriate app store.
- [x] Clicking badges  / CTAs should link to appropriate app stores.

One app store available in country:

- http://localhost:8000/en-US/products/vpn/pricing/?geo=BD

Things to test:

- [x] Should show google play store badge / CTA only.
- [x] Scanning QR Code should go to the Google Play Store.

Test with switch off:

```
./manage.py waffle_switch VPN_WAVE_VII off 
```

- http://localhost:8000/en-US/products/vpn/?geo=AU
- http://localhost:8000/en-US/products/vpn/pricing/?geo=AU

Things to test:

- [x] Waitlist should be displayed for new countries.

VPN download page:

- http://localhost:8000/en-US/products/vpn/download/

Things to test:

- [x] Both iOS and Android download links use new helper and link to the right places.

Note there's no need right now to hide the Apple App Store link based on country for this page. Only from the main landing page.

### CODE REVIEW

File: bedrock/settings/base.py [hunk 1]
```diff
@@ -1790,13 +1790,11 @@ def before_send(event, hint):
     "IM",  # Isle of Man (a British Crown dependency)
     "IO",  # British Indian Ocean Territory
     "JE",  # Jersey (a British Crown dependency)
-    "UK",  # United Kingdom
```
Comment (alexgibson) [inline bedrock/settings/base.py:1793]: Note for reviewer: the new tests added here have uncovered that `UK` and `UM` are in fact unsupported country codes, and have never in fact been returning in our requests. Both of these pre-date VPN coming to bedrock, so I think can safely be removed. According to https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements, `UK` is simply reserved by the United Kingdom, it's not actually used (`GB` is the official code).

**Comment by alexgibson** [comment]
Successful test run: https://github.com/mozilla/bedrock/actions/runs/11366391410

**Comment by alexgibson** [comment]
This is ready for code review but adding do-not-merge until we can get a thumbs up from product team.

**Review by wen-2018 — APPROVED** [review]
I have followed the testing instructions to test out of lists of new countries, they are working as expected for me 👍

File: bedrock/settings/base.py (hunk 1 above)
Comment (janbrasna) [reply bedrock/settings/base.py:1793]: PS: Next time someone edits the list or the whereabouts, the GB comment would also use some love:

> "GB",  # United Kingdom of Great Britain and Northern **Island**

(unless that's an inside joke I'm not following;D…)

**Pull request merged**